llvm/lib/Target/AMDGPU/MCA/AMDGPUCustomBehaviour.h

```diff
@@ -17,6 +17,7 @@
 #ifndef LLVM_LIB_TARGET_AMDGPU_MCA_AMDGPUCUSTOMBEHAVIOUR_H
 #define LLVM_LIB_TARGET_AMDGPU_MCA_AMDGPUCUSTOMBEHAVIOUR_H
 
+#include "Utils/AMDGPUBaseInfo.h"
```

Comment:
I don't quite understand why this is required now but was not required before.

Reply:
Because the "hasModifiersSet" function declared in the header below now accepts an OpName instead of unisgned.